frontend/apps/remark42/app/components/auth/auth.hooks.ts

```diff
@@ -76,11 +76,11 @@ export function useDropdown(disableClosing?: boolean) {
 
     handleChangeIframeSize(dropdownElement);
 
-    const observer = new MutationObserver(() => {
+    const observer = new ResizeObserver(() => {
```

Comment:
I am not sure. Maybe it isn't necessary to have this `observer` and ResizeObserver on `body` will be enough.

Reply:
Unfortunately we need it. I attached the root observer to `body` because I hoped it should see size changes for this thing too. As it turned out it can't spot size change because dropdowns positioned absolutely. I haven't dug into it maybe there is a way to have only one observer. For now I hope this thing will work better than our previous method.

Reply:
Yep, got it